test/test_http.py

```diff
@@ -302,6 +329,58 @@ def test_gzip_trailing_garbage(self):
             data = ydl.urlopen(sanitized_Request(f'http://localhost:{self.http_port}/trailing_garbage')).read().decode('utf-8')
             self.assertEqual(data, '<html><video src="/vid.mp4" /></html>')
 
+    def test_brotli(self):
```

Comment:
Maybe instead of checking the response, how about doing
```suggestion
    @skipUnless(brotli, "brotli support is not installed")
    def test_brotli(self):
```
for all tests that use brotli?

Reply:
wasn't aware of this, thanks